spec/spec_helper.rb

```diff
@@ -77,6 +77,13 @@
   config.asset_host = "http://localhost:3000"
 end
 
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
+
```

Comment:
ShouldaMatchers allow for efficient testing of model relationships, associations, delegations, etc.